md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ArchiveConsumpQueueAPI.java

```diff
@@ -103,24 +103,20 @@ RestWrapper get(
      * passed queueId.
      *
      * @param queueId
-     * @param model
      * @return nothing.
      */
     @RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
-    public
-    @ResponseBody
+    @ResponseBody public
     RestWrapper delete(
-            @PathVariable("id") Long queueId, Principal principal,
-            ModelMap model) {
+            @PathVariable("id") Long queueId, Principal principal) {
         RestWrapper restWrapper = null;
         try {
             archiveConsumpQueueDAO.delete(queueId);
-            //s.delete("call_procedures.DeleteArchiveConsumpQueue", archiveConsumpQueue);
 
             restWrapper = new RestWrapper(null, RestWrapper.OK);
             LOGGER.info("Record with ID:" + queueId + " deleted from ArchiveConsumpQueue by User:" + principal.getName());
         } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)